src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests_ListPatterns.cs

```diff
@@ -1898,9 +1869,8 @@ struct S
         _ = this[i]; // 1, 2
         _ = this[r]; // 3, 4
 
-        _ = this is [0];
-        _ = this is { 1 };
-        _ = this is { 2, ..var rest };
+        _ = this is [ 1 ];
+        _ = this is [ 2, ..var rest ];
```

Comment:
Could you include some tests with designations too? In particular, let's have one verifying the `GetTypeInfo` and `GetDeclaredSymbol` from such designation with a nullable value type input.

Reply:
Updated ListPattern_Symbols_01 to test designations. Does that cover what you had in mind?

Reply:
Yes, that what I had in mind. Thanks